### DISCUSSION TITLE
TreeView SelectionChanged Event

### DISCUSSION THREAD

**Discussion by ParkhutRoman** [microsoft/WindowsAppSDK#1834] [Q&A] [answered]
Will we have SelectionChanged Event back to TreeView? From 1.0.0-preview1 I can only use Event Tapped to get somehow to know ab selected item. TreeViewItem IsSelected Binding dont work, but even if did, i only need to know ab SelectionChanged in tree.

**Comment by andrewleader** [comment, marked as answer]
Hi @ParkhutRoman, since this is a WinUI question, you should ask this in the WinUI repository: https://github.com/microsoft/microsoft-ui-xaml/issues
